src/nsresourced/nsresourcework.c

```diff
@@ -1530,6 +1533,65 @@ static int create_veth(
         return 0;
 }
 
+static int create_tap(
+                int userns_fd,
+                const char *ifname_host,
+                char **altifname_host,
```

Comment:
Not a big matter, but I'd like to insert 'const': `char * const *altifname_host`